es6-template-literals/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```

Comment:
Still looks like other parts of the code were modified, and the conversion wasn't exactly matched:

---

![image](https://user-images.githubusercontent.com/16143771/127083738-7ce6a7ab-56d2-4cda-abed-c15afdd7af23.png)

---

![image](https://user-images.githubusercontent.com/16143771/127083783-b4cea209-72f3-4f1a-8390-1a97ac6995b2.png)